drivers/gps/nrf9160_gps/nrf9160_gps.c

```diff
@@ -685,6 +695,7 @@ static int start(const struct device *dev, struct gps_config *cfg)
 	}
 
 	atomic_set(&drv_data->is_active, 1);
+	atomic_set(&drv_data->timeout_occurred, 0);
```

Comment:
Does it make sense to provide a common cleanup function for atomic variables?

Reply:
They are not set to the same value always. Here it is 1 and 0. In the next place it is 0 and 0.